Source/ACE.Entity/AceObject.cs

```diff
@@ -1220,6 +1220,12 @@ public string DisplayName
             set { SetBoolProperty(PropertyBool.RequiresBackpackSlot, value); }
         }
 
+        public bool UseBackpackSlot
```

Comment:
It really has to be in both places because to do what Mogwai wanted, I don't make inventory a world object until it is used or wielded.   It is only ready only and makes the code much simpler to follow.